### PR TITLE
Entry Point Test

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This adds an entry point test to the integration tests.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
 numpy>=1.13.3
-pandas>=0.23.0
+pandas>=0.23.0,<0.24.0
```
Comment (rwedge) [inline requirements.txt]: Does this library need to restrict itself to pandas 0.23?

File: requirements.txt [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-featuretools>=0.7.0
+featuretools>=0.7.0,<0.12.0
 nltk>=3.4
 tensorflow<=1.14.0
 tensorflow_hub<=0.4.0
 scikit-learn>=0.20.0,<0.21; python_version < '3'
-scikit-learn>=0.20.0; python_version >= '3'
\ No newline at end of file
+scikit-learn>=0.20.0; python_version >= '3'
+python-dateutil==2.8.0
```
Comment (rwedge) [inline requirements.txt]: I think a range would be better, for featuretools I used
`python-dateutil>=2.6.1,<2.8.1`

File: requirements.txt [hunk 3]
```diff
@@ -1,9 +1,10 @@
 numpy>=1.13.3
-pandas>=0.23.0
+pandas>=0.23.0,<0.24.0
 tqdm>=4.19.2
-featuretools>=0.7.0
+featuretools>=0.7.0,<0.12.0
```
Comment (rwedge) [inline requirements.txt]: Same question with restricting featuretools

File: requirements.txt (hunk 1 above)
Comment (jeff-hernandez) [reply requirements.txt]: not necessary, reverted back to 0.23

File: requirements.txt (hunk 2 above)
Comment (jeff-hernandez) [reply requirements.txt]: updated to range

File: requirements.txt (hunk 3 above)
Comment (jeff-hernandez) [reply requirements.txt]: reverted back to >=0.7.0

**Pull request merged**